config/schema/document_types/vehicle_recalls_and_faults_alert.json

```diff
@@ -0,0 +1,111 @@
+{
+  "fields": [
+    "fault_type",
+    "alert_issue_date",
+    "item_type",
+    "manufacturer",
+    "item_model",
+    "serial_number",
+    "build_start_date",
+    "build_end_date"
+  ],
+
+  "allowed_values": {
+    "fault_type": [
+      {
+        "label": "Recall",
+        "value": "recall"
+      },
+      {
+        "label": "Non-urgent fault",
+        "value": "non_urgent_fault"
+      }
+    ],
+
+    "item_type": [
+      {
+        "label": "Vehicle",
+        "value": "vehicle"
+      },
+      {
+        "label": "Baby seat",
+        "value": "baby-seat"
+      },
+      {
+        "label": "Tyres",
+        "value": "tyres"
+      },
+      {
+        "label": "Parts",
+        "value": "parts"
+      },
+      {
+        "label": "Agricultural equipment",
+        "value": "agricultural-equipment"
+      },
+      {
+        "label": "Other accessories",
```

Comment:
Having "other" options makes me wonder if this would be better without "allowed values", allowing any value to be added.  The list feels very non-comprehensive and likely to results in support requests for adding more allowed values in future.

Is this actually only using allowed values because finders require fields with filters to use it?  If so, we should fix that, rather than artificially restrict the list.

The same applies to the list of "manufacturer"s, I think.